src/Facets/SymbiosisFacet.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import { ILiFi } from "../Interfaces/ILiFi.sol";
+import { ISymbiosisMetaRouter } from "../Interfaces/ISymbiosisMetaRouter.sol";
+import { LibAsset, IERC20 } from "../Libraries/LibAsset.sol";
+import { ReentrancyGuard } from "../Helpers/ReentrancyGuard.sol";
+import { SwapperV2, LibSwap } from "../Helpers/SwapperV2.sol";
+import { Validatable } from "../Helpers/Validatable.sol";
+
+/// @title Symbiosis Facet
+/// @author Symbiosis (https://symbiosis.finance)
+/// @notice Provides functionality for bridging through Symbiosis Protocol
+/// @custom:version 1.0.0
+contract SymbiosisFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
+    /// @notice The contract address of the Symbiosis router on the source chain
+    ISymbiosisMetaRouter private immutable symbiosisMetaRouter;
+    address private immutable symbiosisGateway;
+
+    /// Types ///
+    struct SymbiosisData {
+        bytes firstSwapCalldata;
+        bytes secondSwapCalldata;
+        address intermediateToken;
+        address bridgingToken;
+        address firstDexRouter;
+        address secondDexRouter;
+        address relayRecipient;
```

Comment:
Is this the address where the calldata gets executed? If yes, I would suggest to name it "callTo" and to rename the otherSideCalldata to "callData" just to be in line with our other facets